README.md

```diff
@@ -7,13 +7,13 @@ drone-wall
 
 ### Deployment
 
-    $ docker pull scottwferg/drone-wall
-    $ docker run -p 3000:3000 -e API_SCHEME=$API_SCHEME -e API_DOMAIN=$API_DOMAIN \
-        -e API_TOKEN=$API_TOKEN -e API_PORT=$API_PORT scottwferg/drone-wall
+    $ docker pull scottwferg/drone-wall:2.1
+    $ docker run -p 3000:8080 -e API_SCHEME=$API_SCHEME -e API_DOMAIN=$API_DOMAIN \
+        -e API_TOKEN=$API_TOKEN -e API_PORT=$API_PORT -e WALL_PORT=8080 scottwferg/drone-wall:2.1
 
 Drone wall exposes port `3000`. You can map this to whatever you like.
```

Comment:
This makes no sense anymore then.

The `WALL_PORT` is the port that this application is going to bind to within the Docker container - therefore if the `WALL_PORT` is 8080, then exposing port 3000 is completely useless.